public/index.html

```diff
@@ -4,9 +4,11 @@
   src="https://code.jquery.com/jquery-3.5.0.min.js"
   integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
   crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;900&display=swap" rel="stylesheet">
```

Comment:
i was trying to get this to work using an import in the createGlobalStyles component, but to no avail.  Nice work around!

Reply:
You guys are good. I wasn't worrying about getting that specific. I'll have to add this.

Reply:
yeah! I just copy pasted from google fonts, took about 3 mins